ff2mpv.py

```diff
@@ -11,8 +11,9 @@
 def main():
     message = get_message()
     url = message.get("url")
+    time = message.get("time") or 0
```

Comment:
```suggestion
    time = message.get("time")
```

...then, only inject the `--time=TIME` argument if it's present. Performing it unconditionally might have unintended side effects.